oc_config_validate/oc_config_validate/testcases/telemetry_once.py

```diff
@@ -81,7 +84,35 @@ def test100(self):
             f"Expected {self.updates_count} Updates, got: {updates}")
 
 
-class CheckLeafs(SubsOnceTestCase):
+class CountUpdatePaths(SubsOnceTestCase):
+    """Subscribes ONCE and checks the returned update paths.
+
+    This tests that a Subscription reports all Update paths.
+
+    Args:
+        xpaths: List of gNMI paths to subscribe to. Can contain wildcards.
+        paths_count: Number of expected disctinct Update paths.
```

Comment:
Should this be `update_paths_count` instead?

Reply:
Yes! Thanks for spotting it